zerver/openapi/zulip.yaml

```diff
@@ -11746,6 +11746,194 @@ paths:
+                        type: array
+                        description: |
+                          An array of `invites` object.
+                        items:
+                          $ref: "#/components/schemas/Invite"
+                    example:
+                      {
+                        "result": "success",
+                        "msg": "",
+                        "invites":
+                          [
+                            {
+                              email: "example@zulip.com",
+                              expiry_date: null,
+                              id: 1,
+                              invited: 1710606654,
+                              invited_as: 200,
+                              invited_by_user_id: 9,
+                              is_multiuse: false,
+                            },
+                            {
+                              expiry_date: 1711463862,
+                              id: 1,
+                              invited: 1710599862,
+                              invited_as: 400,
+                              invited_by_user_id: 9,
+                              is_multiuse: true,
+                              link_url: "http://localhost:9991/join/6i4324pzbtyvmwv5nwyghoof/",
+                            },
+                          ],
+                      }
+    post:
+      operationId: send-invites
+      summary: Send invites
+      tags: ["invites"]
+      description: |
+        Send invitations to the email addresses of the invitees.
+      requestBody:
+        required: true
+        content:
+          application/x-www-form-urlencoded:
+            schema:
+              type: object
+              properties:
+                invitee_emails:
+                  description: |
+                    The string containing the email addresses of the invitees, separated
+                    by commas or newlines.
+                  type: string
+                  example: example@zulip.com, logan@zulip.com
+                invite_expires_in_minutes:
+                  description: |
+                    The number of minutes before the invitation will expire.This parameter also
+                    accepts `null` which specifies that invitation will never expire. If unspecified,
+                    its default value of `INVITATION_LINK_VALIDITY_MINUTES` will be used.
+
+                    **Changes**: New in Zulip 6.0 (feature level 126). Previously,
+                    `invite_expires_in_days` parameter was used which was used to specify
+                    the duration in days instead of minutes.
+                  oneOf:
+                    - type: integer
+                    - type: string
+                  example: 14400
+                invite_as:
+                  description: |
+                    [Organization-level role](/api/roles-and-permissions) that the invitee
+                    will be assigned to after accepting the invitation.
```

Comment:
This should be a sentence , so add a "The" at the start.

Reply:
I have added another block for mentioning the permission restrictions but I am unsure about the wording.